packages/web-components/src/components/rux-modal/rux-modal.tsx

```diff
@@ -121,18 +144,12 @@ export class RuxModal {
     }
 
     connectedCallback() {
-        setTimeout(() => {
-            const button = this._getDefaultButton()
-            button && button.focus()
-        })
         this._handleModalChoice = this._handleModalChoice.bind(this)
     }
 
-    componentDidLoad() {
-        setTimeout(() => {
-            const button = this._getDefaultButton()
-            button && button.focus()
-        })
+    componentDidRender() {
```

Comment:
Moved all the button focusing to the didRender lifecycle method since it fires with every re-render, thus focusing the button every time the modal is opened, not just the first.